### ISSUE TITLE
Next.js v6 compatibility

### ISSUE THREAD

**Issue by RobbinHabermehl** [bmealhouse/next-redux-saga#8]
Unfortunately `next-redux-saga` doesn't seem to be compatible out of the box with Next.js v6 and `next-redux-wrapper` v2.0.0-beta.4. When configuring the `_app.js` as documented the context isn't being set.

`_app.js`:
```js
import React from 'react';
import nextReduxSaga from 'next-redux-saga';
import withRedux from 'next-redux-wrapper';

import initializeStore from '../store';

class MyApp extends React.Component {
  static async getInitialProps({ Component, ctx }) {
    // we can dispatch from here too
    ctx.store.dispatch({ type: 'FOO', payload: 'foo' });

    const pageProps = Component.getInitialProps ? await Component.getInitialProps(ctx) : {};

    return { pageProps };
  }

  render() {
    const { Component, pageProps } = this.props;
    return (
      <Component {...pageProps} />
    );
  }
}

export default withRedux(initializeStore)(nextReduxSaga(MyApp));
```

This results in the following error:
```
TypeError: Cannot read property 'dispatch' of undefined
```

Using just `next-redux-wrapper` is working, but results in a race condition that often causes sagas not to have been completed before the app was rendered server-side.

**Comment by bmealhouse** [comment]
@RobbinHabermehl - Thanks for reporting this issue. I'm not sure how quickly I will be able to get to it, but we accept PRs if you want to take a look.

**Comment by RobbinHabermehl** [comment]
I've fixed the issue and created PR #9 for this. There's still some work left though, so please feel free to help me out on getting this ready to release! :)

**Comment by bmealhouse** [comment]
I appreciate it! I will try to look at the PR tonight.

**Comment by bmealhouse** [comment]
Closed by #9
